pkg/ccl/streamingccl/streamingest/stream_ingestion_processor_test.go

```diff
@@ -171,6 +171,32 @@ func TestStreamIngestionProcessor(t *testing.T) {
 		require.Nil(t, row)
 		testutils.IsError(meta.Err, "this client always returns an error")
 	})
+
+	t.Run("stream ingestion processor hangs on losing client connection", func(t *testing.T) {
```

Comment:
I haven't reviewed the test yet, and I'm going to think if there is any other way we can structure this.

Reply:
Perhaps instead of testing if the client hangs, we could test if the processor returns an error or not when the client disconnects?